### PR TITLE
[Term Entry] Python String .rindex() function

### PR DESCRIPTION
**Pull request by deenovita**
### Description

- Created a new term entry Python strings .rindex() function.
- Close issue #3332 

### Type of Change

- Adding a new entry.

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-example-course'
+  - 'paths/example-path'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:13]:
```suggestion
  - 'learn-python-3'
  - 'paths/computer-science'
```

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+**`.rindex()`** is used to locate the highest index of the substring within a string variable. ValueError will raise when the substring cannot be found.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:16]:
```suggestion
**`.rindex()`** is used to locate the highest index of the substring within a string variable. A `ValueError` will be raised when the substring cannot be found.
```

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Syntax
+
+The syntax of this function can be shown below :
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:20]:
```suggestion
The syntax of the function takes the following formats:
```

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+```pseudo
+str_variable.rindex('substring')
+
+str_variable.rindex('substring', index_start, index_end)
+```
+
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:27]: there should be a brief summary explaining each parameter

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+## Example
+
+The following examples implement `.rindex()` and will return the index location of substring :
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:30]:
```suggestion
The following examples implement `.rindex()` and will return the index location of substring:
```

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+# Define the string variable
+str_cow = "moo moo"
+
+# Show the results
+print(str_cow.rindex('m'))  # return 4
+
+print(str_cow.rindex('o'))  # return 6
+```
+
+The index of substring m is found on indices 0 and 4. Therefore, index 4 will be returned.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:42]:
```suggestion
This will result in the following output:

```shell
4
6
```

Comment (caupolicandiaz) [reply content/python/concepts/strings/terms/rindex/rindex.md:42]: use a shell block to show the results and delete the output in the comments

File: content/python/concepts/strings/terms/rindex/rindex.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+## Codebyte Example
+
+This example is runnable and demonstrate instance where `.rindex()` will locate the substring between the certain indices :
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/rindex/rindex.md:46]:
```suggestion
This example is runnable and demonstrates an instance where `.rindex()` will locate the substring between the given indices :
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@deenovita I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by deenovita** [comment]
@caupolicandiaz I have updated the entry as suggested. Let me know if there is another thing I need to improve. Thank you

**Review by caupolicandiaz — APPROVED** [review]
@deenovita This is ready for a second review.

**Comment by deenovita** [comment]
> @deenovita This is ready for a second review.

Thank you!

**Review by SSwiniarski — APPROVED** [review]
@deenovita, @caupolicandiaz Looks good to me.

**Comment by deenovita** [comment]
> @deenovita, @caupolicandiaz Looks good to me.

Thank you!

**Pull request merged**